setup.py

```diff
@@ -142,6 +142,7 @@ def check_previous():
     'google-api-python-client>=1.5.0, <1.6.0',
     'oauth2client>=2.0.2, <2.1.0',
     'PyOpenSSL',
+    'google-cloud-dataflow',
```

Comment:
@fenglu-g @Fokko Do we really need the 'google-cloud-dataflow' library here as what I can see is we are using Google's discovery based API for all google cloud related commands?

Please correct me if I am wrong or if I misinterpreted something.

Reply:
The reason I am asking this is due to the issues with dependencies as stated in this 2 Jira issues:
* https://issues.apache.org/jira/browse/AIRFLOW-2343
* https://issues.apache.org/jira/browse/AIRFLOW-2266

Reply:
I see, fine with removing this for now. We'll fix it on the google-data-flow side.

Reply:
Thanks @fenglu-g . I shall submit a PR removing 'google-cloud-dataflow' and will mention you so that you can approve it.